src/vehicle.cpp

```diff
@@ -2954,6 +2954,22 @@ bool vehicle::part_flag( int part, const vpart_bitflags flag ) const
     }
 }
 
+void vehicle::connect_appliance_to_neighbours()
+{
+    map &here = get_map();
+
+    for( const tripoint trip : here.points_in_radius( sm_pos, 1 ) ) {
```

Comment:
```suggestion
    for( const tripoint &trip : here.points_in_radius( sm_pos, 1 ) ) {
```

This loop added in the previous PR causes an error in the build matrix in clang 12 macOS build.